cmplibrary/src/androidTest/java/com/sourcepoint/cmplibrary/campaign/CampaignManagerImplTest.kt

```diff
@@ -178,4 +181,20 @@ class CampaignManagerImplTest {
 
         cm.metaDataResp.assertNull()
     }
+
+    @Test
+    fun `GIVEN_an_expired_GDPR_and_a_valid_CCPA_campaign_DELETE_only_the_GDPR_consent`() {
+        val json = JSONObject("v7/expired_gdpr_valid_ccpa.json".file2String())
+        appContext.storeTestDataObj(json.toList())
+        ds.deleteGdprConsent()
+        appContext.spEntries().toList().find { it.first.contains("GDPR") }.assertNull()
+    }
+
+    @Test
+    fun `GIVEN_an_expired_CCPA_and_a_valid_GDPR_campaign_DELETE_only_the_GDPR_consent`() {
+        val json = JSONObject("v7/expired_ccpa_valid_gdpr.json".file2String())
+        appContext.storeTestDataObj(json.toList())
+        ds.deleteCcpaConsent()
+        appContext.spEntries().toList().find { it.first.contains("CCPA") }.assertNull()
```

Comment:
I think we should assert that `GDPR` campaign is `not null` in this case, since we want to make sure that only `CCPA` campaign was deleted

Reply:
same logic as before but different campaign